dart/lib/src/protocol/app.dart

```diff
@@ -76,7 +76,7 @@ class App {
     }
 
     if (startTime != null) {
-      json['app_start_time'] = startTime.toIso8601String();
+      json['app_start_time'] = startTime!.toIso8601String();
```

Comment:
is this required because its non final? so it could mutate from one thread to another

Reply:
The compiler complains otherwise. I'm not quite sure tho why. It's final and also the compiler should be able to infer that it's not null.